src/main.py

```diff
@@ -28,7 +30,8 @@ def post_message(pull_requests):
 
     if text:
         payload = {"blocks": text}
-        requests.post(slack_webhook, data=json.dumps(payload))
+        if not DEBUG:
+            requests.post(slack_webhook, data=json.dumps(payload))
```

Comment:
thoughts on pushing this to cloudwatch as opposed to slack?

Reply:
yeah, `DEBUG` is a log level... this looks to be more of a "dry run" kind of thing, a toggle for actually doing something vs just telling you what it would do. i'd recommend setting up the logger and using that facility, along with a `DRY_RUN` env.

Reply:
What you are asking isn't entirely clear. The purpose of the project is to publish Open PRs to slack so that the operators are seeing it. Seems like you are looking for additional logging which I am not opposed to.